### PR TITLE
Added update command to file distribution

### PR DESCRIPTION
**Pull request by mkot02**
Updating file distribution is implemented in pulp API,
but wasn't added to pulp-cli.

This feature allows to change publication and base path
of existing distribution.

fixes #8145
https://pulp.plan.io/issues/8145

### CODE REVIEW

File: tests/scripts/test_file_distribution.sh [hunk 1]
```diff
@@ -23,7 +23,9 @@ expect_succ pulp file repository sync --name "cli_test_file_repository"
 expect_succ pulp file publication create --repository "cli_test_file_repository"
 PUBLICATION_HREF=$(echo "$OUTPUT" | jq -r .pulp_href)
 
-expect_succ pulp file distribution create --name "cli_test_file_distro" \
+expect_succ pulp file distribution create --name "cli_test_file_distro" --base-path "wrong_path" 
+expect_succ pulp file distribution update \
+  --name "cli_test_file_distro" \
```
Comment (mdellweg) [inline tests/scripts/test_file_distribution.sh:28]: Can you add nullifying the publication somewhere here?

**Review by mdellweg — COMMENTED** [review]
Codewise nice and concise.
Just asking for one more test.

File: tests/scripts/test_file_distribution.sh (hunk 1 above)
Comment (mkot02) [reply tests/scripts/test_file_distribution.sh:34]: Added test. See commit message for more detailed description

**Review by mdellweg — APPROVED** [review]
Thank you!

**Pull request merged**